src/services/MakeStreamZip.php

```diff
@@ -80,7 +80,14 @@ public function __destruct()
      */
     public function getFileName(): string
     {
-        return 'export.elabftw.zip';
+        $this->idArr = explode(' ', $this->idList);
+        if (count($this->idArr) == 1){
+            $this->Entity->setId((int) $this->idArr[0]);
+            $this->Entity->getPermissions();
+            return $this->getBaseFileName() . ".zip";
+        }else{
```

Comment:
spacing!!

Reply:
fixed (I need to install a formatter for php :))